### DISCUSSION TITLE
can Playcover run in the background? like Android emulator

### DISCUSSION THREAD

**Discussion by CrazyEig** [PlayCover/PlayCover#1527] [Ideas]
Hello, can Playcover run in the background? I wrote a Python script to automatically play a game, but the script only works when the game window is in the foreground. Additionally, simulating mouse clicks causes my mouse to move to the game window, which prevents me from doing other work. Is it possible to add a feature ，such as I send coordinates (x, y) to Playcover, and Playcover automatically clicks in the background? Thank you.

**Comment by MEMORIEmusic** [comment]
Are you on macOS 15 beta, by any chance?

**Reply by CrazyEig** [reply]
I‘m 13.4.1 ，can 15 beta do that？

**Reply by MEMORIEmusic** [reply]
No. But it used to work for me on Ventura and Sonoma. Stopped working that way in macOS 15 beta. I was guessing that it might be the cause, but apparently not.

**Reply by CrazyEig** [reply]
I'm now on Ventura，how can you do that on Ventura?

**Reply by MEMORIEmusic** [reply]
Idk? It used to work that way. Are you in the nightly version? Perhaps something changed in the newer nightlies.
